http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSAlgebra.scala

```diff
@@ -9,7 +9,9 @@ trait AKSAlgebra[F[_]] {
   /** Creates an app and polls it for completion */
   def createAndPollApp(params: CreateAKSAppParams)(implicit ev: Ask[F, AppContext]): F[Unit]
 
-  // TODO (TOAZ-230): delete, stop, start
+  def deleteApp(params: DeleteAKSAppParams)(implicit ev: Ask[F, AppContext]): F[Unit]
+
+  // TODO (TOAZ-230): stop, start
```

Comment:
TOAZ-230 is just for delete app -- don't think we have a ticket for stop/start yet, maybe just remove the comment.